Es6 complete project

luccamicale opened the pull request:
I complete the project using ES6 with Javascript. Now my website is very complete,  and the code is very tidy and clean. 
I created four different parts inside the folder "modules" 
.Functions
.Variables
.Luxon
.constructor

luccamicale committed:
The project it is complete

luccamicale committed:
Modify things in README.md

luccamicale committed:
fix linters

luccamicale committed:
linters fix

luccamicale committed:
fin lintersss

mikethreels commented on index.js:
```diff
@@ -0,0 +1,90 @@
+  aumento = 0;
+});
+
+function showAddBook() {
+  document.getElementById('add-book').style.display = 'flex';
+  document.getElementById('list-book').style.display = 'none';
+  document.getElementById('contact').style.display = 'none';
+}
+
+function showContact() {
+  document.getElementById('contact').style.display = 'flex';
+  document.getElementById('list-book').style.display = 'none';
+  document.getElementById('add-book').style.display = 'none';
+}
```
- [x] according to the requirements we are required to use ES6 syntax which includes arrow functions instead of the `function`  keyword

Abenezer-Tilahun replied:
Fixed.

mikethreels commented on modules/Functions.js:
```diff
@@ -0,0 +1,9 @@
+import { timeAndDate } from './variables.js';
+import { DateTime } from './luxon.js';
+
+export default function dateAndTimeNow() {
+  setInterval(() => {
+    const date = DateTime.now().toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+    timeAndDate.innerHTML = date;
+  }, 0);
```
- [x] according to the requirements we are required to use ES6 syntax which includes arrow functions instead of the `function`  keyword

Abenezer-Tilahun replied:
Fixed

mikethreels reviewed (changes requested):
Hi Lucca, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luccamicale committed:
I fixed all things

luccamicale commented:
I fixed all problems in my code. Now, the code is perfect and must be approved.

.I changed the function for functions arrows in the three lines that ask me the last reviewer.

luccamicale commented:
![arrows2](https://user-images.githubusercontent.com/107897361/193855587-94a2f9bb-49c7-4e66-9be6-e1a93a7d162e.PNG)
![arrows1](https://user-images.githubusercontent.com/107897361/193855596-543322c3-3410-4989-b662-58b6e3bb0e3b.PNG)

Abenezer-Tilahun reviewed (approved):
Hi @luccamicale      ,

### Status: Approved ✔️ ✔️ ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏
## Optional Suggestion

- [ ] Kindly Fix the MIT License in your README file the link is broken. 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
